kubos/utils/git_utils.py

```diff
@@ -55,20 +55,36 @@ def get_latest_tag(tag_list):
 def fetch_tags(repo):
     origin = repo.remotes.origin
     logging.info('Checking for newer releases...') #Tags mark new KubOS releases
-    origin.fetch(tags=True)
+    origin.fetch()
 
 
-def checkout_and_update_version_file(tag, repo):
+def checkout_branch_update_version(branch, repo):
+    logging.info("Checking out branch '%s'" % branch)
+    logging.warning("Branch '%s' of the Kubos Source has not been marked as stable yet. Proceed with caution." % branch)
```

Comment:
Are we planning on having stable branches in the future?  Seems like our tags are guaranteed stable and that's it.  So maybe something like "Warning: Kubos branches are not guaranteed to be stable. Proceed with caution" might be more accurate?